src/modules/r.rs

```diff
@@ -0,0 +1,101 @@
+use regex::Regex;
```

Comment:
Go's module is named `src/modules/golang.rs` while config is named `src/configs/go.rs`. If we apply that here we get `src/modules/rlang.rs` and the corresponding config becomes `src/config/r.rs`.

Reply:
Hmm I think Go is popularly known as Go lang, whereas other modules are not called pythonlang, rustlang, etc... I admit though that `r` is quite a bit short and cryptic. Single letter language could have this suffix such as dlang, clang, etc... What is your reasoning? is this align on your thinking?

Reply:
> I admit though that `r` is quite a bit short and cryptic. Single letter language could have this suffix such as dlang, clang, etc...

You've pretty much said it all - adding lang suffix to avoid too short names following golang's precedent.

Reply:
Module name has been changed to `rlang`.